src/Apps/Artist/Routes/Consign/Components/Utils/getConsignSubmissionUrl.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Construct a url that encodes various tracking parameters needed to be sent to
+ * segment from the force side of the app.
+ */
+export function getConsignSubmissionUrl(props: {
+  contextPath: string
+  subject: string
+}) {
+  return `/consign/submission?contextPath=${props.contextPath}/consign&subject=${props.subject}`
```

Comment:
I don't think we are passing any values of `contextPath` into this function that _require_ URL encoding, but to save someone in the future some work, we could encode the value of `contextPath` when building this URL.